index.md

```diff
@@ -64,11 +64,11 @@ Often it is best when you first contact an organization to assume that they don
 
 ### Asking Others to Help {#ask}
 
-Consider asking for help understanding the problem and communicating it. Someone might be able to help you understand the website or assistive technologies. Someone might help you communicate to the website owners. For example, an advocacy group.
+Consider asking friends, family, or colleagues for help understanding the problem you are observing and communicating it to the website owner. For example, the issue may be related to the website or any assistive technologies you are using. You can also ask other to help you communicate with the website owner. For example, an advocacy group.
```

Comment:
Ask whom for help? Communicate it to whom? This sections seems too abbreviated to follow. Also, there are three distinct points that seem very unclear:
- you can ask someone for help understanding the problem
- you can ask someone for help understanding your own technology
- you can ask someone for help interacting with the website owner